aemcmc/dists.py

```diff
@@ -50,30 +52,93 @@ def multivariate_normal_rue2005(rng, b, Q):
     return u + v
 
 
-# A and omega are assumed to be diagonal
-def multivariate_normal_cong2017(rng, A, omega, phi, t):
-    """
-    Sample from a multivariate normal distribution with a structured mean
-    and covariance matrix, as described in Example 4 [page 17] of [1]. This
-    algorithm is suitable for high-dimensional regression problems and the
-    runtime scales linearly with the number of regression coefficients.
+def multivariate_normal_cong2017(
+    rng: RandomStream,
+    A: TensorVariable,
+    omega: TensorVariable,
+    phi: TensorVariable,
+    t: TensorVariable,
+) -> TensorVariable:
+    r"""
```

Comment:
Nitpick: docstring should start after the `"""`

Reply:
I think this is mainly a preference thing. The current way is still correct according the `numpydoc` convention and is used all over numpy and scipy source code.

Reply:
When you start a repo every reasonable choice is certainly a question of *preference*, in an existing codebase it is a matter of *consistency*.

Reply:
I understand your concern but `dists.py` is the first module commited into the codebase and if you look at the other functions in it, they use the same docstring convention. That is about as consistent as can be in this case. Using the suggested preference would introduce an inconsistent docstring style in the module. So the point made here is a bit contradictory.

Reply:
There are only two choices here: change the docstrings in every other files in the repository, or press DEL twice in this file. Whatever you think is best.

Reply:
All the `aesara-devs` projects follow [the code and documentation standards of Aesara](https://aesara.readthedocs.io/en/latest/dev_start_guide.html#documentation-and-docstrings).  In the case of docstrings, we use NumPy's format&mdash;more or less.  (The link in our documentation is broken; it should be [this](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard).)

We have some legacy docstrings in the Aesara repo that go against this standard, but those need to be fixed.